competitions/cloud-cover/benchmark-tutorial.ipynb

```diff
@@ -0,0 +1,2586 @@
+    "\n",
+    "1. Demonstrate how to explore and work with the data\n",
+    "2. Provide a basic framework for building a model\n",
+    "3. Demonstrate how to package your work correctly for submission\n",
+    "\n",
+    "This notebook is a version of the benchmark [blog post](https://www.drivendata.co/blog/cloud-cover-benchmark/) posted on the competition website, with some small tweaks to run directly in the planetary computer hub.\n",
+    "\n",
+    "You can either expand on and improve this benchmark, or start with something completely different! Let's get started."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<hr>\n",
+    "\n",
+    "Satellite imagery is critical for a wide variety of applications from disaster management and recovery, to agriculture, to military intelligence. Clouds present a major obstacle for all of these use cases, and usually have to be identified and removed from a dataset before satellite imagery can be used. Improving methods of identifying clouds can unlock the potential of an unlimited range of satellite imagery use cases, enabling faster, more efficient, and more accurate image-based research.\n",
+    "\n",
+    "**In this challenge, your goal is to detect cloud cover in satellite imagery**. \n",
+    "\n",
+    "The challenge uses publicly available satellite data from the [Sentinel-2 mission](https://sentinel.esa.int/web/sentinel/missions/sentinel-2), which captures wide-swath, high-resolution, multi-spectral imaging. For each tile, data is separated into 13 spectral bands capturing the full visible spectrum, near-infrared, and infrared light. Feature data is shared through Microsoft's [Planetary Computer](https://planetarycomputer.microsoft.com/).\n",
+    "\n",
+    "The ground truth labels for cloud cover are part of a new dataset created by [Radiant Earth Foundation](https://www.radiant.earth/). After the competition ends, the labels dataset will be made publicly available to contribute to ongoing learning and advancement in the field.\n",
+    "\n",
+    "In this post, we'll cover:\n",
+    "\n",
+    "- [Exploring the data](#explore-the-data)\n",
+    "- [Preparing the data](#split-data)\n",
+    "- [Following the code execution submission format](#code-exec)\n",
+    "- [Building the benchmark model](#build-model)\n",
+    "- [Fitting the benchmark model](#fit-model)\n",
+    "- [Generating a code execution submission](#generate-submission)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%%capture\n",
+    "!pip install pandas_path pytorch_lightning cloudpathlib loguru typer segmentation_models_pytorch"
```

Comment:
I'm deploying an update today with pytorch-lighting, and segmentation-models-pytorch should already be available. So I think those two can be dropped.

Reply:
Sounds good, just let me know when that's ready (I'm testing it out now, and looks like the current version still needs them installed)